apps/src/lab2/views/components/PredictSummary.tsx

```diff
@@ -53,32 +52,31 @@ const PredictSummary: React.FunctionComponent = () => {
   };
 
   return (
-    <InstructorsOnly>
-      <div className={moduleStyles.predictSummaryContainer}>
-        <LinkButton
-          href={summaryUrl}
-          text={commonI18n.viewStudentResponses()}
-          size={'xs'}
-          type={'secondary'}
-          color={'black'}
-        />
-        {responseCount !== null && numStudents !== null && (
-          <div className={moduleStyles.responses}>
-            <div className={moduleStyles.responseIcon}>
-              <i className="fa fa-user" />
-            </div>
-            <div>
-              <span className={moduleStyles.responseCount}>
-                {responseCount}/{numStudents}
-              </span>
-              <span className={moduleStyles.responseLabel}>
-                {commonI18n.studentsAnswered()}
-              </span>
-            </div>
+    <div className={moduleStyles.predictSummaryContainer}>
+      {responseCount !== null && numStudents !== null && (
+        <div className={moduleStyles.responses}>
+          <div className={moduleStyles.responseIcon}>
+            <i className="fa fa-user" />
```

Comment:
I don't have a preference just curious when it's preferred to use a vanilla icon versus using the FontAwesomeV6 component

Reply:
Honestly this was just a product of copying over code a few times, and I think the original code predates the component 😄 I have found that sometimes I can't style thee component exactly how I want.